Processes/DevRel-Publication-Process.md

```diff
@@ -1,30 +1,34 @@
-# Process: Publish New or Updated Content to the Dev Portal
+# Process: Write, Edit, Update Content using GitHub
```

Comment:
This is a rather long name, but regardless, this procedure is not really about "publishing" to the docs portal. It's about "contributing" content to it. Therefore, a more appropriate name would be "Docs Contribution Process" or similar.